packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoInput/index.js

```diff
@@ -1,58 +1,75 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useIntl } from 'react-intl';
 import PropType from 'prop-types';
 import { CarouselInput, CarouselSlide, CarouselActions } from '@strapi/design-system/CarouselInput';
 import { IconButton } from '@strapi/design-system/IconButton';
 import { Box } from '@strapi/design-system/Box';
 import Plus from '@strapi/icons/Plus';
+import LogoModalStepper from '../LogoModalStepper';
+import { SIZE, DIMENSION } from '../../utils/constants';
 
 const LogoInput = ({ customLogo, defaultLogo }) => {
   const { formatMessage } = useIntl();
+  const [isDialogOpen, setIsDialogOpen] = useState(true);
```

Comment:
Maybe we should rename this state to `isModalOpen`?

Reply:
would make sense with the modal component name! thanks!